index.html

```diff
@@ -211,5 +211,34 @@ <h1 class="about-head">About me</h1>
           </li>
         </ul>
       </section>
+      <section class="contact-form">
+        <h3 class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        <form class="form" action="https://formspree.io/f/xvonwgga" method="post">
+          <input class="input-text" type="text" placeholder="Full name" maxlength="30" required>
+          <input class="input-text" type="text" placeholder="Email Address" required>
+          <textarea class="input-message" name="message" d=""i cols="30" rows="10" maxlength="500">Write me something</textarea>
```

Comment:
- [ ] Great job implementing requested changes by the previous reviewer, there's one thing you've not implemented yet. According to the project requirements. All input fields should have the `required` attribute to activate the validation on submit. In your case, the `required` is missing in the textarea field. Kindly add that to it. Also, I would recommend you to remove the `Write me something` text inside the <textarea> tag and make it placeholder instead so that, the required attribute can work.